### DISCUSSION TITLE
plugn version compatibility?

### DISCUSSION THREAD

**Discussion by perpetuallight** [dokku/dokku#6189] [Q&A] [answered]
Hello,

I have a client owned platform hosting an older version of docker and dokku or at least dokku. Recently the client implemented a break/fix without the fix and it appears that the ```plugn``` module was the target. I have no frame of reference as to why it was targeted, at least not in the security context. But, obviously, it has rendered dokku functions useless. The tricky part is that while the client owns the platform, dokku was implemented by the initial dev that started the contract. Which means that they are not likely to take ownership of the break/nofix.

So my question is this, will any version of ```plugn``` be compatible with the older version of dokku if I were to only reinstall the ```plugn``` module in accordance with the repo instructions? Or are they version dependent? And if so is there a record/table of compatibility version matches?

**Comment by josegonzalez** [comment, marked as answer]
There is no table with any compatibility guarantees, but you can check the `Makefile` for the version of dokku in question. The top of the file has versions we tested against in CI, and that should be enough to figure out which version of `plugn` to use.

**Comment by perpetuallight** [comment]
Perfect, thank you.
